tools/make_hostdiscover.cpp

```diff
@@ -0,0 +1,61 @@
+/*
+Copyright Soramitsu Co., Ltd. 2016 All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+#include <json.hpp>
+
+#include <fstream>
+#include <infra/config/iroha_config_with_json.hpp>
+#include <consensus/connection/connection.hpp>
+#include <thread>
+#include <util/ip_tools.hpp>
+
+using Api::DiscoverRequest;
+using Api::Peer;
+
+int main(int argc, char* argv[]) {
+    std::vector<std::string> defaultHosts {};
+    auto trustedHosts = config::IrohaConfigManager::getInstance().getTrustedHosts(defaultHosts);
+    if (trustedHosts.empty()) {
+        std::cout << "Config section for trusted hosts was not found or it is empty." << std::endl;
+        return 1;
+    }
+
+    std::ofstream hostsDiscovered("discovered.txt");
```

Comment:
@Warchant stdout is spammed with grpc logs, config read logs and so on:
```
1490863700 DEBUG [config] load json is {
  "database_path":"/tmp/iroha_ledger",
  "java_class_path":"java_tests",
  "java_class_path_local":"smart_contract/java_tests",
  "java_library_path":"lib",
  "java_library_path_local":"build/lib",
  "java_policy_path":"jvm/java.policy.txt",
  "concurrency": 0,
  "max_faulty_peers" : 1,
  "pool_worker_queue_size": 1024,
  "http_port": 1204,
  "grpc_port": 50051,
  "active_start": false,
  "trusted_hosts": [
    "127.0.0.1",
    "172.17.0.2",
    "172.17.0.3",
    "172.17.0.4",
    "172.17.0.5",
    "188.130.155.39/8"
  ]
}

1490863700 INFO [connection] Host discover service
1490863700 INFO [connection] message: discovery
1490863700 INFO [connection] response: IaDYg2mKk2u1JQcKw5PL4qgdDKVKyj+l74rvu7xrS1k=
127.0.0.1 IaDYg2mKk2u1JQcKw5PL4qgdDKVKyj+l74rvu7xrS1k=
1490863700 INFO [connection] Host discover service
1490863700 INFO [connection] message: discovery
1490863701 INFO [connection] Host discover service
1490863701 INFO [connection] message: discovery
1490863702 INFO [connection] Host discover service
1490863702 INFO [connection] message: discovery
1490863703 INFO [connection] Host discover service
1490863703 INFO [connection] message: discovery
1490863704 WARNING [Networks larger than /16 are not supported for now] 
```
So I think file is better.